src/hooks/useKeyboardNavigation.js

```diff
@@ -57,10 +116,74 @@ const useKeyboardNavigation = ({
     ].filter(Boolean);
   }, []);
 
+  const focusNextEmojiListGroup = (columnIndex = 0) => {
+    const currentEmojiGroup = getCurrentEmojiListGroup();
+    const nextEmojiGroup = currentEmojiGroup.nextSibling;
+
+    if (nextEmojiGroup) {
+      nextEmojiGroup.children[columnIndex].firstChild.focus();
+
+      const categoryName = nextEmojiGroup.getAttribute('data-name');
+      activateCategoryByName(categoryName);
+    }
+  };
+
+  const focusPrevEmojiListGroupOnLastItem = () => {
+    const currentEmojiGroup = getCurrentEmojiListGroup();
+    const prevEmojiGroup = currentEmojiGroup.previousSibling;
+
+    if (
+      prevEmojiGroup &&
+      prevEmojiGroup.lastChild &&
+      prevEmojiGroup.lastChild.firstChild
+    ) {
```

Comment:
I believe our babel setup supports optional chaining (`?.`)

Reply:
I get the following error ..
![image](https://user-images.githubusercontent.com/7828909/151702039-871dfc6f-73a0-438f-9819-8f5f492e75f3.png)

According to 
[eslint-optional-chaining-error-with-vscode](https://stackoverflow.com/questions/61628947/eslint-optional-chaining-error-with-vscode)

we should update the eslint to eslint@^7.5

Reply:
sure. I'll take it